great_expectations/execution_engine/sqlalchemy_batch_data.py

```diff
@@ -264,28 +264,39 @@ def _create_temporary_table(  # noqa: C901, PLR0912, PLR0915
         # Similar message may be needed in the future for Trino backend.
         elif dialect in (GXSqlDialect.TRINO, GXSqlDialect.CLICKHOUSE):
             logger.warning(
-                f"GX has created permanent view {temp_table_name} as part of processing SqlAlchemyBatchData, which usually creates a TEMP TABLE."  # noqa: E501
+                f"GX has created permanent view {temp_table_name}"
+                " as part of processing SqlAlchemyBatchData, which usually creates a TEMP TABLE."
             )
             stmt = f"CREATE TABLE {temp_table_name} AS {query}"
         elif dialect == GXSqlDialect.AWSATHENA:
             logger.warning(
-                f"GX has created permanent TABLE {temp_table_name} as part of processing SqlAlchemyBatchData, which usually creates a TEMP TABLE."  # noqa: E501
+                f"GX has created permanent TABLE {temp_table_name}"
+                " as part of processing SqlAlchemyBatchData, which usually creates a TEMP TABLE."
             )
             stmt = f"CREATE TABLE {temp_table_name} AS {query}"
-        elif dialect == GXSqlDialect.ORACLE:
-            # oracle 18c introduced PRIVATE temp tables which are transient objects
-            stmt_1 = f"CREATE PRIVATE TEMPORARY TABLE {temp_table_name} ON COMMIT PRESERVE DEFINITION AS {query}"  # noqa: E501
-            # prior to oracle 18c only GLOBAL temp tables existed and only the data is transient
-            # this means an empty table will persist after the db session
-            stmt_2 = f"CREATE GLOBAL TEMPORARY TABLE {temp_table_name} ON COMMIT PRESERVE ROWS AS {query}"  # noqa: E501
         # Please note that Teradata is currently experimental (as of 0.13.43)
         elif dialect == GXSqlDialect.TERADATASQL:
-            stmt = f'CREATE VOLATILE TABLE "{temp_table_name}" AS ({query}) WITH DATA NO PRIMARY INDEX ON COMMIT PRESERVE ROWS'  # noqa: E501
+            stmt = (
+                f'CREATE VOLATILE TABLE "{temp_table_name}" AS ({query})'
+                " WITH DATA NO PRIMARY INDEX ON COMMIT PRESERVE ROWS"
+            )
         elif dialect == GXSqlDialect.VERTICA:
             stmt = f"CREATE TEMPORARY TABLE {temp_table_name} ON COMMIT PRESERVE ROWS AS {query}"
         else:
             stmt = f'CREATE TEMPORARY TABLE "{temp_table_name}" AS {query}'
+
         if dialect == GXSqlDialect.ORACLE:
+            # oracle 18c introduced PRIVATE temp tables which are transient objects
+            stmt_1 = (
+                f"CREATE PRIVATE TEMPORARY TABLE {temp_table_name}"
+                f" ON COMMIT PRESERVE DEFINITION AS {query}"
+            )
+            # prior to oracle 18c only GLOBAL temp tables existed and only the data is transient
+            # this means an empty table will persist after the db session
+            stmt_2 = (
+                f"CREATE GLOBAL TEMPORARY TABLE {temp_table_name}"
+                f" ON COMMIT PRESERVE ROWS AS {query}"
+            )
```

Comment:
this looks like a feature unrelated to the mypy change - if this is going to be included, we should test it

Reply:
It is because of the mypy rule change.
The only thing not related are the formatting changes.

https://github.com/great-expectations/great_expectations/pull/10091/files#diff-80bb3fa3777f40e701e2d78e2ab270456bf1f7f88ec61c331fcaff00f2219ee7

Reply:
thanks! looked net new, but i see that its just shuffled around